src/main/java/ui/components/pickers/PickerLabel.java

```diff
@@ -10,16 +15,17 @@
 // for use with LabelPickerDialog
 public class PickerLabel extends TurboLabel {
 
-    private final LabelPickerUILogic labelPickerUILogic;
+    private final LabelPickerDialog presenter;
```

Comment:
This class should be cleaned up a bit also. `final` fields, removal of dead/commented code, javadocs, etc.